Eatery Blue/Eatery Blue/UI/Carousel/CarouselMoreLoadingCardsView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CarouselMoreLoadingCardsView.swift
+//  Eatery Blue
+//
+//  Created by Antoinette Marie Torres on 4/25/23.
+//
+
+import UIKit
+
+class CarouselMoreLoadingCardsView: UIView {
+
+    let stackView = UIStackView()
+    let imageView = UIView()
+    let titleLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setUpSelf()
+        setUpConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setUpSelf() {
```

Comment:
just do this stuff in `init` you don't have to create a function name `setupSelf`